Sofa/Sofa/League Matches/League/LeagueHeaderView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  LeagueHeaderView.swift
+//  Sofa
+//
+//  Created by Akademija on 28.03.2025..
+//
+import UIKit
+import SnapKit
+import SofaAcademic
+
+class LeagueHeaderView: UITableViewHeaderFooterView {
+
+    static let reuseIdentifier = "LeagueHeaderView"
+
+    private let leagueView = LeagueView()
+
+    var league: League? {
+        didSet {
+            if let league = league {
+                leagueView.update(with: league)
+            }
+        }
+    }
```

Comment:
Ova ti vrijednost zapravo ne treba, jer jedino što želiš je proslijediti league vrijednost u view, tako da napravi configure ili update metodu koju ćeš pozivati izvana.

Nek ti u Cellu i općenito u Viewu budu storane vrijednost samo one koje ti nužno trebaju za view. U ovom slučaju cell treba samo proslijediti vrijednosti za update viewa.